### DISCUSSION TITLE
"No such file or directory" Problem with loading Models in Fooocus

### DISCUSSION THREAD

**Discussion by julgold2000** [lllyasviel/Fooocus#3641] [Q&A] [answered]
Hey guys, I'm trying to install/use a Model in Fooocus with Google Colab. It worked all perfectly until I was trying to use a new model.

So before I run Fooocus I added this code:

!wget -O /content/Fooocus/models/checkpoints/realismEngineSDXL_v30VAE.safetensors https://civitai.com/models/152525/realism-engine-sdxl

I also had trouble in running it in first place. When I was about to start Fooocus he showed:
ValueError: No such file or directory: "/content/Fooocus/models/checkpoints/realismEngineSDXL_v30VAE.safetensors"

So Fooocus started but I couldn't find the Model.
At the same step I was upgrading my Google Colar to Premium and activated the A100 GPU.

When I used then the normal GPU it started without showing that issue. So I got successfully in Fooocus and could find the new model. I activate it and was trying to make something. But then nothing happened. Its not loading or generating anything.

I checked it in the console and it says the same:
ValueError: No such file or directory: "/content/Fooocus/models/checkpoints/realismEngineSDXL_v30VAE.safetensors"

Would be awesome if I could receive some help.
Thanks.

**Comment by poor7** [comment, marked as answer]
You need to change "https://civitai.com/models/152525/realism-engine-sdxl" to "https://civitai.com/api/download/models/293240".
